.github/workflows/plugin-test.yml

```diff
@@ -0,0 +1,33 @@
+on: [push, pull_request]
+
+name: Test Plugin
+
+jobs:
+  load:
+    name: Plugin Loads
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: "Downloading IPFS"
+        id: "download"
+        run: |
+          IPFS_VERSION="$(go list -f '{{.Module.Version}}' github.com/ipfs/go-ipfs)"
```

Comment:
What exactly does this do? Get the last tagged release? Or the current master?

It's a bit weird to use go here before actually installing it, but I guess as long as it works, it's fine...

Reply:
It gets the go-ipfs version from this repo's `go.mod` file. We need that so we can install the correct go version... I'll add a comment.